packages/apps/docs/src/scripts/utils.ts

```diff
@@ -1,12 +1,12 @@
-import type { Content, Root } from 'mdast-util-from-markdown/lib';
+function hasChildren(content: any): content is {
+  children: any[];
+} {
+  return 'children' in content;
+}
 
-export const getTypes = <T>(
```

Comment:
what happened with all the any?

Reply:
this is an internal import which broke on `moduleResolution` bundler mode, and we shouldn't use them anyway, those types are re-exports from the main package `mdast`

Reply:
and btw this is still typesafe I wrote a guard for that reason, the thing is `Root` and `Content` don't overlap so probably they are some other  type we should use here

Reply:
found a better solution there are no any